crates/biome_cli/src/reporter/terminal.rs

```diff
@@ -93,7 +93,7 @@ impl fmt::Display for SummaryDetail {
             })
         } else {
             fmt.write_markup(markup! {
-                " No fixes needed."
+                " No fixes were applied."
```

Comment:
I'm not sure whether we should use `No fixes applied` or `No fixes were applied`?

Reply:
Let's use the past tense. We use it already in the other sentences, e.g. `Checked ...`, `Formatted ...`, etc.

Reply:
> Let's use the past tense. We use it already in the other sentences, e.g. `Checked ...`, `Formatted ...`, etc.

Technically they're both past tense and in passive voice, but this is not the question I guees?

I think the problem here is whether we should use a whole sentence (`No fixes were applied`) or a short message (`No fixes applied`). The short one aligns with the previous message `No fixes needed`. 🤔

Reply:
Honestly? I dunno 😅

Reply:
> Honestly? I dunno 😅

I decide to change it to `No fixes applied`:

- It aligns with the previous message;
- Other messages are also not whole sentences: `Checked 1 file in <TIME>`, `Found 1 error.`. They all lack subjects.